sdcm/coredump.py

```diff
@@ -83,11 +83,13 @@ def update(self,
 class CoredumpThreadBase(Thread):
     lookup_period = 30
     upload_retry_limit = 3
+    # If exceptions will happen MAX_COREDUMP_THREAD_EXCEPTIONS times - fail the thread and don't run it forever
 
-    def __init__(self, node: 'BaseNode', max_core_upload_limit: int):
+    def __init__(self, node: 'BaseNode', max_core_upload_limit: int, max_coredump_thread_exceptions: int = 10):
         self.node = node
         self.log = SDCMAdapter(node.log, extra={"prefix": self.__class__.__name__})
         self.max_core_upload_limit = max_core_upload_limit
+        self.max_coredump_thread_exceptions = max_coredump_thread_exceptions
```

Comment:
Can we make it a class variable?

Reply:
This variable is changed during unit tests. Class variable is protected and can't be changed

Reply:
fixed